### PR TITLE
Tomorrows technology

### PR DESCRIPTION
**Pull request by mtrich**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 15 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 15 points 
- [ ] Appropriate use of Semantic Elements: 15 points
- [ ] Effective use of box model: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 15 points

**Checkpoint 2**
- [ ] Effective use of Flexbox layout: 15 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 15 points 
- [ ] Effective use of CSS Grid: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Site is hosted on github: 15 points
- [ ] Linter passing: 10 points
- [ ] The student demonstrated mastery of HTML/CSS and Responsive Web Design by building a website of professional caliber: 15 points

**Checkpoint 3**
- [ ] Use of HTML: 15 points
- [ ] Use of CSS: 15 points
- [ ] Use of JQuery: 15 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Linter passing: 10 points
- [ ] The application has a professional look: 15 points
- [ ] The application functions as it should: 15 points

### CODE REVIEW

File: 04week/nasa/css/reset.css
Comment (situlillian) [inline 04week/nasa/css/reset.css:1]: you should not have previous commits in this branch! make sure you're making a new branch from gh-pages each time

File: 04week/tomorrows-technology/index.html
Comment (situlillian) [inline 04week/tomorrows-technology/index.html:15]: 👍

**Review by situlillian — APPROVED** [review]
you're kicking ass! it looks great. good layout and styling

**Comment by situlillian** [comment]
can you please give me write access to this repo?

**Comment by DinhH** [comment]
@mtrich please don't merge your own pull request!

**Pull request merged**
